src/anomalib/metrics/pimo/binary_classification_curve.py

```diff
@@ -26,7 +26,7 @@
 logger = logging.getLogger(__name__)
 
 
-def _binclf_one_curve(scores: ndarray, gts: ndarray, threshs: ndarray) -> ndarray:
+def _binary_classification_one_curve(scores: ndarray, gts: ndarray, threshs: ndarray) -> ndarray:
```

Comment:
I wonder if 'binary_classification' can be reduced here as well? For the time being, we all work on binary classification, no?

Reply:
We have imports like this in Anomalib `from torchmetrics.classification import BinaryPrecisionRecallCurve`. So I personally feel we could keep it as `binary_classification_curve` to remain consistent with torchmetrics